pages/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <title>Gale Internets</title>
+  <link href="../stylesheets/styles.css" rel="stylesheet"/>
+  <link href="https://fonts.googleapis.com/css?family=Maven+Pro" rel="stylesheet"/>
+</head>
+<header>
+  <nav>
+    <ul>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a href="portfolio.html">Portfolio</a>
+      </li>
+      <li>
+        <a href="code-journal.html">Journal</a>
+      </li>
+    </ul>
+  </nav>
+  <div class="white-space-bottom"></div>
```

Comment:
instead of using `<div class="white-space-bottom"></div>` it may be better to give a bottom margin to your nav